File: src/C/1010.c
/*
 * 1010 - Simple Calculate
 * Author: Otavio
 * Date: 26/04/2024
 */
#include <stdio.h>

struct product
{
	int id;
	int q;
	double val;
};

int
main()
{
	struct product a, b;
	scanf("%d %d %lf", &(a.id), &(a.q), &(a.val));
	scanf("%d %d %lf", &(b.id), &(b.q), &(b.val));

	double total = (a.q*a.val) + (b.q*b.val);
	printf("VALOR A PAGAR: R$ %.2lf\n", total);
	return 0;
}

File: src/JS/1010.js
/*
 * 1010 - Simple Calculate
 * Author: Otavio
 * Date: 31/08/2024
 */
const input = require('fs').readFileSync('/dev/stdin', 'utf8');
const lines = input.split('\n');

const first = lines.shift().split(' ');
const second = lines.shift().split(' ');

first.shift(); //p1 id
const firstAmount = parseInt(first.shift());
const firstPrice = parseFloat(first.shift());

let total = firstAmount*firstPrice;

second.shift(); //p2 id
const secondAmount = parseInt(second.shift());
const secondPrice = parseFloat(second.shift());

total += secondAmount*secondPrice;

console.log(`VALOR A PAGAR: R$ ${total.toFixed(2)}`);
